src/components/SavedFilters.vue

```diff
@@ -39,17 +39,26 @@
     return allOptions
   })
 
+  const filterInRoute = computed<SavedSearchFilter>(() => ({
+    state: filter.flowRuns.state.name ?? [],
+    flow: filter.flows.id ?? [],
+    deployment: filter.deployments.id ?? [],
+    workPool: filter.workPools.name ?? [],
+    tag: filter.flowRuns.tags.name ?? [],
+    startDate: filter.flowRuns.expectedStartTimeAfter != undefined ? String(filter.flowRuns.expectedStartTimeAfter) : undefined,
+    endDate: filter.flowRuns.expectedStartTimeBefore != undefined ? String(filter.flowRuns.expectedStartTimeBefore) : undefined,
+  }))
+
+  const { value: myCustomDefaultFilter } = useCustomDefaultFlowRunsFilter()
+  watch(filterInRoute, (newValue) => {
+    if (myCustomDefaultFilter.value !== null && isEmptyFilter(newValue)) {
+      setFilters(myCustomDefaultFilter.value)
+    }
+  }, { immediate: true })
```

Comment:
Won't this mean a user can never get back the the prefect defined default view of one week? Maybe that's intended. 

I'm wondering if it would be better to preserve the options in the menu but `onMount` check if there is a user defined default and select that. But then let the user choose the "Default view" if they wanted to. 

Maybe we should change "Default View"  to show up as something else 🤔 its just a filter for the last week. Maybe we should label it as such.

Reply:
it's not totally explicit, but there's a slight distinction between _selecting_ the "default view" and navigating to `/flow_runs`. On navigation to `/flow_runs`, the `isSameFilter` ignores checking the start/end dates in the route and calls it "default". On selecting "default view", the route gains the implicit start/end dates that define the oneWeekSavedSearch default filter. 

I'm taking advantage of that so that an empty route maps to a custom default filter if one exists else fallback to current behavior. Otherwise, the "default view" option is still pickable since it has its own route query params it adds. 

> Maybe we should change "Default View" to show up as something else 🤔 its just a filter for the last week. Maybe we should label it as such.

Yeah I'm toying with that right now. atm thinking of maybe a default tag or just some `(default)` kind of text on the current default option and replacing what currently reads as "default view" as "Last week" or something.

Reply:
sounds good, thanks for clearing that up 👍